phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -544,7 +556,28 @@ private MetaDataMutationResult updateCache(PName tenantId, String schemaName, St
 
         return result;
     }
-    
+
+    public MetaDataMutationResult updateCache(String schemaName) throws SQLException {
+        return updateCache(schemaName, false);
+    }
+
+    public MetaDataMutationResult updateCache(String schemaName, boolean alwaysHitServer) throws SQLException {
+        long clientTimeStamp = getClientTimeStamp();
+        PSchema schema = null;
+        try {
+            schema = connection.getMetaDataCache().getSchema(new PTableKey(null, schemaName));
```

Comment:
No, we wouldn't need a schema per tenantId. With namespace support, is all data still stored in the same physical table for all views, even if the views have different schemas? For example, given the following:

    CREATE VIEW s1.a AS SELECT * FROM t WHERE k=1;
    CREATE VIEW s2.b AS SELECT * FROM t WHERE k=2;
    CREATE VIEW s3.c AS SELECT * FROM t WHERE k=3;

Would the following query still return all rows across all three views?

    SELECT * FROM t;